src/Microsoft.AspNetCore.Mvc.Abstractions/Microsoft.AspNetCore.Mvc.Abstractions.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <Import Project="..\..\build\common.props" />
+  <Import Project="..\..\build\sourcelink.props" />
```

Comment:
Still haven't tried it but `@` not `%` for the item groups. And, I should have included more quotes:
``` xml
<Import Project="..\..\build\sourcelink.props" Condition=" '@(OtherProjectsToPack)' != '@(ProjectsToPack)' "/>
```